app/controllers/SingleRecordController.scala

```diff
@@ -67,57 +67,60 @@ class SingleRecordController @Inject() (
             updatedAnswersWithCountryOfOrigin.set(CommodityCodeUpdatePage(recordId), record.comcode)
           )
         _                                  <- sessionRepository.set(updatedAnswersWithAll)
-      } yield {
-        val detailsList = SummaryListViewModel(
-          rows = Seq(
-            TraderReferenceSummary.row(record.traderRef, recordId, NormalMode),
-            GoodsDescriptionSummary.row(record.goodsDescription, recordId, NormalMode),
-            CountryOfOriginSummary.row(record.countryOfOrigin, recordId, NormalMode),
-            CommodityCodeSummary.row(record.comcode, recordId, NormalMode),
-            StatusSummary.row(record.declarable)
+      } yield
+        if (record.toReview) {
+          Redirect(routes.ReviewReasonController.onPageLoad(recordId).url)
```

Comment:
Bit wary of us doing this in here, because its the kind of thing we want to avoid doing in our controllers. Ideally the navigator needs to figure out it needs to show that page instead of this one?

Reply:
Have a look at the review reason controller, this has been undone - It's a single redirect that happens on page load as well... I think pulling that out to the navigator might be overkill + it's based on a link and nothing else.